### PR TITLE
[ADD]  Express server with user management endpoints

### PR DESCRIPTION
**Pull request by Charbrown08**
code review
Thomk Corporation

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const port = 3000;
+
+app.use(express.json());
+
+let users = [];
+
+// Endpoint to get all users
+app.get('/users', (req, res) => {
+    res.json(users);
+});
```
Comment (Charbrown08) [inline index.js:13]: Hey , This line is bad!

File: index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// Endpoint to delete a user by id
+app.delete('/users/:id', (req, res) => {
+    const { id } = req.params;
+    users = users.filter(user => user.id !== id);
+    res.status(204).send();
+});
+
+app.listen(port, () => {
+    console.log(`Server is running on http://localhost:${port}`);
+});
```
Comment (Hilder09) [inline index.js:31]: OK

**Review by Hilder09 — APPROVED** [review]

File: index.js
Comment (laflopet) [inline index.js:1]: Bien

**Review by laflopet — APPROVED** [review]
Bien

**Pull request merged**